test/app_web/live/app_live_test.exs

```diff
@@ -806,4 +806,50 @@ defmodule AppWeb.AppLiveTest do
     {:ok, view, _html} = live(conn, "/")
     assert render_hook(view, "filter-tags", %{"key" => "t", "value" => "t"})
   end
+
+  test "select tag when enter pressed", %{conn: conn} do
+    {:ok, _tag1} =
+      Tag.create_tag(%{person_id: 0, text: "tag1", color: "#FCA5A5"})
+
+    {:ok, _tag2} =
+      Tag.create_tag(%{
+        person_id: 0,
+        text: "enter_tag_selected",
+        color: "#FCA5A5"
+      })
+
+    {:ok, view, _html} = live(conn, "/")
+
+    assert render_keydown(view, "add-first-tag", %{"key" => "Enter"}) =~
+             "enter_tag_selected"
+
+    assert render_submit(view, :create, %{text: "tag enter pressed"})
+
+    assert render(view) =~ "tag enter pressed"
+    assert render(view) =~ "enter_tag_selected"
+  end
```

Comment:
Good test :) 

You could take it just a *tiny bit* step further and actually **create** the item. Then, call `Item.get_item(id, true)` (true will preload the tags) and `assert` if the `tag` you've created is associated with it.

After that, it's absolutely perfect 🤩

Reply:
OK! I'll do this, but how can I know the id of the item? since it was created by the `render_submit`?

Reply:
Since `render_submit` only returns the view, the easiest way would be to use `Item.list_items` and find the item with the same text you've created. Because each unit test is contained, there shouldn't be any performance issues :)

Reply:
Done! I did a little different solution, let me know what you think!